capabilities/communication/send-templated-email/maps/mailgun.test.ts

```diff
@@ -1,7 +1,28 @@
 import { sendTemplatedEmailTest } from './send-templated-email';
 
-sendTemplatedEmailTest('mailgun', {
-  from: 'demo@demo.superface.org',
-  to: 'demo@superface.ai',
-  templateId: 'station-test',
-});
+sendTemplatedEmailTest(
+  'mailgun',
+  {
+    from: 'demo@demo.superface.org',
+    to: 'demo@superface.ai',
+    templateId: 'station-test',
+  },
+  {
+    beforeRecordingSave: recordings => {
+      recordings.forEach(recording => {
+        recording.path = recording.path.replace(
+          process.env.MAILGUN_DOMAIN as string,
+          'example.com'
+        );
+      });
+    },
+    afterRecordingLoad: scopes => {
```

Comment:
Discussed with @martinalbert an option to have beforeRecordingSave and afterRecordingLoad same interface. Which means working with definitions.json directly.

I hope it will be clearer to users that they can do transformation of the file they see stored, and also keeps nock implementation hidden from them.